packages/core/src/tracing/opentelemetry-instrumentations/tedious.js

```diff
@@ -18,14 +18,6 @@ module.exports.init = () => {
     instrumentation.enable();
   }
 };
-exports.transform = otelSpan => {
-  // NOTE: This assignment is necessary to display the database name in the UI.
-  // In the backend, for OpenTelemetry, the service name is based on the OpenTelemetry span attribute service.name.
-  if (otelSpan.attributes && 'db.name' in otelSpan.attributes) {
-    otelSpan.resource._attributes['service.name'] = otelSpan.attributes['db.name'];
```

Comment:
`otelSpan.resource._attributes` no longer exists in v2.
I tested tedious manually. Works fine. I am not sure why we had to add this earlier, but seems like we don't need this anymore.